user/configure-device.rst

```diff
@@ -0,0 +1,132 @@
+Installing LEDE and connecting it to OpenWISP2
+==============================================
+
+This page will guide you through the installation of `openwisp-config <https://github.com/openwisp/openwisp-config>`_ on an
+`OpenWRT <https://openwrt.org/>`_ device. For simplicity, we will be installing it in a Virtual Machine running
```

Comment:
change to something like "on a device which supports OpenWRT or LEDE (its recent fork which we prefer because more up to date)." Put the phrase after (For simplicity ecc) on a subsequent line with a blank line preceding it so we can make it stand out.